baseclasses/data_transformations/uvvisconcentrationdetection.py

```diff
@@ -45,7 +45,15 @@ class UVvisConcentrationDetection(Analysis, PlotSection):
 
     blank_substraction = Quantity(
         type=Reference(UVvisData.m_def),
-        a_eln=dict(component='ReferenceEditQuantity'))
+        a_eln=dict(component='ReferenceEditQuantity'),
+        description='If this calibration is computed with internal blank substraction, the user can provide a blank'
+                    'which gets substracted of each new measurement when later using this calibration.'
+                    'Please note that this blank also affects the minimum and maximum peak values of the calibration.')
+
+    blank_substraction_peak_value = Quantity(
+        type=np.dtype(np.float64),
+        default=0.0,
+        description='This value is computed automatically if a blank substraction is given and is 0 otherwise.')
```

Comment:
I decided to have this extra field to have a faster way of reading the peak value in the uvvis getConcentrationData. Following a ref there seems to be quite hard.

Reply:
Apart from that everything should be done like discussed in our call this morning.